src/main/java/org/broadinstitute/hellbender/utils/io/IOUtils.java

```diff
@@ -229,6 +229,20 @@ public static File writeTempResource(Resource resource) {
         return temp;
     }
 
+    /**
+     * Create a resource from a path and a relative class, and write it to a temporary file.
+     * @param resourcePath Relative or absolute path to the class.
+     * @param relativeClass Relative class to use as a class loader and for a relative package.
```

Comment:
Javadoc `@param` tags usually come at the end, rather than in the middle of the comment text.

Reply:
Doh.